src/pages/ImportData.js

```diff
@@ -392,7 +393,27 @@ const ImportData = () => {
     };
   };
 
-  const importTdrExport = (namespace, name) => {
+  const loadWdsUrl = useCallback((workspaceId) => {
+    return Ajax()
+      .Apps.listAppsV2(workspaceId)
+      .then(resolveWdsUrl)
+      .then((url) => {
+        return url;
+      });
+  }, []);
+
+  const importTdrExport = (workspace) => {
+    if (workspace.cloudPlatform === 'Azure') {
+      return async () => {
+        // find wds for this workspace
+        const wdsUrl = await loadWdsUrl(workspace.workspaceId);
+        const wdsDataTableProvider = new WdsDataTableProvider(workspace.workspaceId, wdsUrl);
+
+        // call importsnapshot
+        wdsDataTableProvider.importTdr(workspace.workspaceId, snapshotId);
```

Comment:
Struggling to figure out where to inspect the response for a 403 and return an error with a helpful message - currently since the 403 isn't an exception, it just swallows the error.

Reply:
I think you'd have change the `importTdr` function - instead of returning `res.json()`, it could return `res` and then you could check the status code here.

Reply:
or, looks like the `fetchOk` function helps here?